open-company/open-company-web
# Mobile share

### Review after #336 

Card: https://trello.com/c/nKpWC0GP

To test on your mobile device:
- setup an org with slack
- make sure it has the bot
- load a board
- tap the vertical ellipsis
- tap on Share
- [x] share via slack
- [x] share via url (check the copy button actually copy the url)
- [x] share via email
